include/enet/protocol.h

```diff
@@ -10,7 +10,7 @@
 enum
 {
    ENET_PROTOCOL_MINIMUM_MTU             = 576,
-#ifdef __WIIU__
+#if defined(__WIIU__) || defined(__3DS__)
    ENET_PROTOCOL_MAXIMUM_MTU             = 1400,
```

Comment:
Did you actually see ENet negotiating MTUs of this size? We have our [default MTU set to 900](https://github.com/cgutman/enet/blob/c07aa74c60ce9896c49a15f63fa3b207fb96c29e/include/enet/enet.h#L206) and think Sunshine should use that too.

Reply:
This didn't seem to be an issue with Sunshine, but it did cause connection issues with gamestream.
I'm still having trouble with Sunshine, but it doesn't seem like it stems from this.